packages/scss/src/components/_form.scss

```diff
@@ -139,7 +140,7 @@
 		clear: both;
 	}
 
-	&:hover {
+	&:not(.is-disabled):hover {
```

Comment:
&:not(:disabled):not(.is-disabled) à rajouter ?

Reply:
This `.is-disabled` class is used on a parent of the `<input>`, not on the input itself so we cannot target the `disabled` attribute.